src/Silverback.Integration/Diagnostics/IntegrationLogEvents.cs

```diff
@@ -522,6 +522,15 @@ public static class IntegrationLogEvents
             GetEventId(78, nameof(ErrorProcessingOutbox)),
             "Error occurred processing the outbox.");
 
+        /// <summary>
+        ///     Gets the <see cref="LogEvent" /> representing the log that is written when a message is
+        ///     not valid (see <see cref="ValidationMode"/>).
+        /// </summary>
+        public static LogEvent MessageNotValid { get; } = new(
```

Comment:
@BEagle1984 I completely agree, you're right is better to have dedicated event Ids for producer and consumer.

Reply:
@BEagle1984 I removed the ProducerEndpointBuilderTests and InvalidMessageNotProcessed because are not necessary since a dedicated exception in thrown.